### DISCUSSION TITLE
What is the status of MPD on Android?

### DISCUSSION THREAD

**Discussion by madorian** [MusicPlayerDaemon/MPD#2006] [Q&A] [answered]
I'm a little confused;)

Is MPD on Android capable of playing local files?

It says it can only play streams, but is this still correct? 

How can I track the development of MPD on Android?

**Comment by amrx06** [comment]
I use MPD 23.14 every day on an android TV box. Files are on a network storage, I didn't test local files.
It's very buggy, crashes 1-2 times a day. I can't find why and I can't get a log file even with this in mpd.conf 

`log_level				"verbose"
log_file				"/storage/emulated/0/Android/data/org.musicpd/files/mpd.log.txt"`

**Reply by MaxKellermann** [reply]
If MPD crashes, write a bug report with a backtrace. If you don't do that, the bug will probably never be fixed.

**Reply by amrx06** [reply]
Correct.
How do I get backtrace and how do I access to log file on android?

**Reply by bartkl** [reply]
@amrx06 Connect your phone to a PC/laptop with `adb` (Android Debug Bridge) installed. Make sure your phone has Developer Mode enabled, and Enable USB Debugging from the Developer Options. Then, run `adb logcat` to start seeing the system log. Among those log lines you'll fine those you're looking for as well.

**Comment by MaxKellermann** [comment]
Huh, this is not a bug report.

**Comment by MaxKellermann** [comment, marked as answer]
> Is MPD on Android capable of playing local files?

Yes.

> It says it can only play streams, but is this still correct?

Where does it say that?

> How can I track the development of MPD on Android?

By looking at the git commits.
